perfkitbenchmarker/providers/profitbricks/flags.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2016 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from perfkitbenchmarker import flags
+
+
+# Locations
+US_LAS = 'us/las'
+DE_FKB = 'de/fkb'
+DE_FRA = 'de/fra'
+
+# Zones
+ZONE_1 = 'ZONE_1'
+ZONE_2 = 'ZONE_2'
+
+flags.DEFINE_string('profitbricks_config',
+                    os.getenv('PROFITBRICKS_CONFIG',
+                              '~/.config/profitbricks-auth.cfg'),
+                    ('Path to config file containing your email and password. '
+                     'Can also be set via $PROFITBRICKS_CONFIG environment '
+                     "variable.\n(File format: email:password)"))
+
+flags.DEFINE_enum('location',
```

Comment:
Please also prefix this with "profitbricks_" to make it clear that it only sets the location for this provider. If there is a natural way to express both an availability zone and a location in a single string I'd prefer using that to having two separate flags.